src/import/index.ts

```diff
@@ -189,8 +194,8 @@ async function _importProfileGroup(dataSource: ProfileDataSource): Promise<Profi
       return toGroup(importFromInstrumentsDeepCopy(contents))
     }
 
-    if (/^(Stack_|Script_|Obj_)\S+ log opened \(PC\)\n/.exec(contents.firstChunk())){
-      console.log("Importing as Papyrus profile")
+    if (/^(Stack_|Script_|Obj_)\S+ log opened \(PC\)\n/.exec(contents.firstChunk())) {
```

Comment:
All changes unrelated to hermes are from running `npm run prettier`. Might be worth adding `npm run prettier --check` to the ci.yml so PRs fail if committers have not run the formatter

Reply:
This was already part of the repo (though using an eslint plugin instead of `prettier --check` for reasons I don't remember)

But I broke it 3 years ago! PR to fix that is here: #454 

I'm going to wait until this merges though to avoid you dealing with annoying merge conflicts